### DISCUSSION TITLE
Book PDF section numbering format

### DISCUSSION THREAD

**Discussion by jpmorr** [quarto-dev/quarto-cli#1888] [Q&A]
Typically when I use latex for creating documents like books/reports I would set a different page number format for frontmatter and the main contents (e.g. Preface would be on page (i), List of Figures on page (ii), etc, while Chapter 1 would start on page 1).

How do you achieve this affect when using `quarto` to render to pdf? In `latex` it's simply a case of adding `\pagenumbering{roman}`or `\pagenumbering{arabic}` before each section begins? Is this a `pandoc` limitation?

I've noticed the ToC, ToF are already using roman numerals (but don't start at (i) as the cover page and cover back are numbered) but the preface then starts at 1.

**Comment by eeholmes** [comment]
The Pandoc latex templates are fully customizable.

See the great-wave example in [quarto_titlepages](https://nmfs-opensci.github.io/quarto_titlepages/)

Specifically, the [before-body.tex](https://github.com/nmfs-opensci/quarto_titlepages/blob/main/titlepages/great-wave/before-body.tex) file where I redefine how the title page is created and on line 6, tell LaTeX start the page numbers at 1. Although this is a scrbook document class and I am in the frontmatter so this is actually causing the table of contents to start at i.

**Reply by jpmorr** [reply]
@eeholmes Thanks, yes. I've just been trying out [quarto_titlepages](https://nmfs-opensci.github.io/quarto_titlepages/) over the weekend and was figuring it may be possible to add a new preface and/or acknowledgement page using an `\include{preface}` in the `before-body.tex`. As I already have these in the `qmd` format and also want them to render in the `HTML` version, I'd prefer to use the `\include` hack.

I've actually just posted a question trying to figure out how to add `edition` and `isbn` on to the title page like the way you have used `title`, `subtitle`, etc.

**Reply by mcanouil** [reply]
You need to learn Pandoc template syntax for short.
